frontend/app/services/zuora/Rest.scala

```diff
@@ -31,6 +31,16 @@ object Rest {
   case object Cancelled extends SubscriptionStatus
   case object Expired extends SubscriptionStatus
 
+  /*
+   * Zuora product catalogue entry: https://knowledgecenter.zuora.com/BC_Developers/REST_API/B_REST_API_reference/Catalog
+   */
+  case class ProductCatalog(products: List[Product]) {
```

Comment:
should the return generic type be uniformed in these 2 methods? i.e. both `Seq` or bothe `Set`